cmd/test.go

```diff
@@ -273,7 +273,7 @@ func processWatcherUpdate(ctx context.Context, testParams testCommandParams, pat
 
 	err := pathwatcher.ProcessWatcherUpdate(ctx, paths, removed, store, filter.Apply, testParams.bundleMode,
 		func(ctx context.Context, txn storage.Transaction, loaded *initload.LoadPathsResult) error {
-			if len(loaded.Files.Documents) > 0 {
+			if len(loaded.Files.Documents) > 0 || removed != "" {
```

Comment:
If say a policy file was removed and data stayed the same we'd up re-writing the same data to the store I think 🤔 . Should be fine but something to think about.

Reply:
good point, it could have been sharper if we checked whether the "removed" was data or not
Since store.Write() seem to be idempotent in terms of end-result, it probably wouldn't affect much
Btw, While debugging, I saw multiple "Watching for changes ..." (aka processWatcherUpdate calls) for a single `mv` command, those can be investigated too to see why there are multiple different set of signals being caught

Reply:
>   I saw multiple "Watching for changes ..." (aka processWatcherUpdate calls) for a single mv command, those can be investigated too to see why there are multiple different set of signals being caught

It's possible temp files were created under the hood that triggered some events.